e2e-testing/cypress/e2e/12-data-view/data-view.cy.js

```diff
@@ -20,4 +20,9 @@ describe('Automated Tests for Post Filters in Data View', () => {
     dataViewFilterFunctions.click_data_view_btn();
     dataViewFilterFunctions.check_post_filter_by_status();
   });
+
+  it('Steps to delete post on data view', () => {
```

Comment:
@shakirandagire  This is a test to delete a post, added inside a filter code block. It's out of place. Move it elsewhere, where we have add post, edit post etc.